helm/templates/deployment.yaml

```diff
@@ -57,11 +58,6 @@ spec:
             httpGet:
               path: /liveness
               port: http
-          readinessProbe:
-            initialDelaySeconds: {{ .Values.initialDelaySeconds }}
-            httpGet:
-              path: /liveness
-              port: http
```

Comment:
what is the problem with the readiness probe?

Reply:
on #19 `readinessProbe` was added. I don't know if it was ever tested, but the behavior at the moment was that the pod was never ready and kept on restarting. removing the `readinessProbe` solved this behavior due to readiness defaulting to success state.